src/main/scala/com/gd/twitteranalytics/reports/ActiveUserDataFrameReport.scala

```diff
@@ -0,0 +1,29 @@
+package com.gd.twitteranalytics.reports
+
+import ActiveUserReportProcessor._
+import TwitterReportGenerator._
+
+object ActiveUserDataFrameReport extends TwitterReport {
```

Comment:
There are still copy past in the TwitterReport implementations...

Reply:
What are principal differences between the implementations that you see?

Reply:
> There are still copy-pasts in the TwitterReport implementations...

Can you please pin point this one ?

Reply:
Don't get me wrong, I'd like to help you here, but it takes one just to look and compare three different files and find similarities/repeated patterns - which is a pretty straightforward task.
So, please, put a bit more effort and "surprise" me with your design and refactoring skills :)